src/components/2-Longer.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { action } from '@storybook/addon-actions';
-import SlateTranscriptEditor from './index.js';
+import SlateTranscriptEditor from '.';
```

Comment:
is this shorthand for import './index.js'?

Reply:
In short: basically yes.

Long answer:

In this case, `index.js` became `index.ts`. VSCode/the TS language server doesn't let you use the sytax `import x from 'file.ts'`, so there are two options here:

1. `import SlateTranscriptEditor from '.'`
2. `import SlateTranscriptEditor from './index'`